Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -448,10 +448,42 @@ - (void)_uninstallScreenTimeWebpageController
     if (!_screenTimeWebpageController)
         return;
 
+    if (_screenTimeBlurredSnapshot) {
+        [_screenTimeBlurredSnapshot removeFromSuperview];
+        _screenTimeBlurredSnapshot = nil;
+    }
```

Comment:
Optional suggestion, you could make this more concise. If `_screenTimeBlurredSnapshot` was already `nil`, then it's a no-op.

```objc
[std::exchange(_screenTimeBlurredSnapshot, nil) removeFromSuperview];
```